tests/integration.rs

```diff
@@ -464,6 +464,13 @@ fn setup_ovs_dpdk_guests(
     (child1, child2)
 }
 
+#[derive(PartialEq)]
+enum HotplugType {
+    Acpi,
+    VirtioMem,
+    NoHotplug,
+}
```

Comment:
I wouldn't bother creating this enum given all you want is to know if we need to hotplug or not. And we know the snapshot/restore integration test uses virtio-mem exclusively. Simply use a boolean instead.